### ISSUE TITLE
[Homematic] Binary RPC Error

### ISSUE THREAD

**Issue by tbail** [openhab/openhab2-addons#3165]
After updating to latest nightly 2.3.0 1195 the homaticbinding / gateway if broken with the following error.

```
2018-01-24 10:40:06.544 [ERROR] [nicator.server.BinRpcResponseHandler] - No BinX signature
java.io.UnsupportedEncodingException: No BinX signature
        at org.openhab.binding.homematic.internal.communicator.message.BinRpcMessage.validateBinXSignature(BinRpcMessage.java:91) [214:org.openhab.binding.homematic:2.3.0.201801232258]
        at org.openhab.binding.homematic.internal.communicator.message.BinRpcMessage.<init>(BinRpcMessage.java:79) [214:org.openhab.binding.homematic:2.3.0.201801232258]
        at org.openhab.binding.homematic.internal.communicator.server.BinRpcResponseHandler.run(BinRpcResponseHandler.java:44) [214:org.openhab.binding.homematic:2.3.0.201801232258]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
```

The error is confirmed by other openHAB users. Both users are using homegear in different version. With this error the gateway is not able to connect and therefore no devices are usable.

## Expected Behavior
The error should not occur and the gateway should connect to homegear

## Current Behavior
As described above. Connection to homegear is not established 

## Steps to Reproduce (for Bugs)
Just install homematic binding with homegear in version 0.8.0 1979 (nightly) or 0.7.14-1563 (stable)

## Your Environment
OpenHAB version 2.3.0 1195 on Raspberry Pi 3 based on openhabian with jessie
Homegear 0.8.0 1979

**Comment by martinvw** [comment]
@gerrieg

**Comment by gerrieg** [comment]
@martinvw 
I have updated my development environment to the latest master. I can't start openhab in Eclipse anymore. Now I have removed everything, created a new clone and reinstalled Eclipse. Openhab hangs somewhere on startup, if I shutdown, a hanging Java thread is still available.
I don't know what to do anymore. Were there any serious changes that could cause the problem?
(latest OracleJDK, macOS High Sierra)

**Comment by martinvw** [comment]
I saw multiple similar reports, for example these

https://github.com/openhab/openhab-core/issues/247
https://github.com/openhab/openhab-core/issues/248
https://github.com/eclipse/smarthome/issues/4955

I will also install Eclipse on my Mac, to check it out.

**Comment by martinvw** [comment]
I have the same problems :'-(

**Comment by tbail** [comment]
Error is resolved by OH 2.3.0 1197 in combination with homegear 0.80-1988

So this issue will be closed
